src/win/fs.c

```diff
@@ -1667,6 +1667,70 @@ void fs__closedir(uv_fs_t* req) {
   SET_REQ_RESULT(req, 0);
 }
 
+/* This function is called when fs__stat_impl_from_path
+*  CreateFileW fails with the code ERROR_ACCESS_DENIED (0x5)
+*  It's a fallback when the user doesn't have the permission
+*  To open a handle to the desired file, this queries less
+*  Information, but still have the most important ones.
+*
+*  These are the values we can access in this method:
+*  - File Attributes
+*  - Creation Time
+*  - Last Access Time
+*  - Last Write Time
+*  - File Size
+*/
+INLINE static int fs__stat_nohandle(WCHAR* path, uv_stat_t* statbuf) {
+  int ret;
+  WIN32_FIND_DATAW find_data;
+  HANDLE handle;
+
+  handle = FindFirstFileW(path, &find_data);
+
+  if (handle == INVALID_HANDLE_VALUE)
+    ret = -1;
```

Comment:
```suggestion
    return -1;
```
(and remove the `else`)

Reply:
Changes committed